scripts/fix_bashrc.sh

```diff
@@ -44,9 +44,13 @@ export PYTHONPATH=/opt/ros/melodic/lib/python3/dist-packages/:$PYTHONPATH
 export PYTHONPATH=$PYTHONPATH:/opt/ros/melodic/lib/python2.7/dist-packages/
 
 # set prompt text/color based on type of container
+parse_git_branch() {
+     git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/ (\1)/'
+}
+
 if [[ ! -v DEPLOYMENT_DOCKER ]]; then 
-	export PS1="🐳  \[\033[36m\]\h\[\033[32m\] (dev) \[\033[33;1m\]\w\[\033[m\] (\$(git branch 2>/dev/null | grep '^*' | colrm 1 2)) "
+	export PS1="\[\033[32m\]\h🐳 \[\033[36m\]\u@dev\[\033[m\]:\[\033[33;1m\]\w\[\033[m\]\$(parse_git_branch) $ "
 else
-	export PS1='🐳  \[\033[36m\]\h\[\e[0;49;91m\] (deploy) \[\033[33;1m\]\w\[\033[m\] '
+	export PS1="\[\e[0;49;91m\]\h🐳 \[\033[36m\]\u@deploy\[\033[m\]:\[\033[33;1m\]\w\[\033[m\]\$(parse_git_branch) $ "
```

Comment:
Please no git branch in deploy docker.

Reply:
I also don't think `\u` is helpful since it is always `root` in a docker anyway

Reply:
I don't think the branch parsing is working for submodules under `/src`, just tested by adding this block to `.bashrc` in a running container. So perhaps best to remove in both cases